deploy/operator.yaml

```diff
@@ -650,20 +650,9 @@ rules:
   verbs:
   - get
   - list
-  - update
-  - watch
   - patch
-- apiGroups:
```

Comment:
why are we removing things from operator.yaml?
This will break pflex deployment.

Reply:
@JacobGros, nothing is removed from operator.yaml. This manifest was manually edited to add support for Powerflex driver. I made the required changes to get these generated in operator.yaml manifest. Please check the file completely and let know if something is removed. Since these are generated now the ordering would change